src/libraries/AccountAccessParser.sol

```diff
@@ -332,6 +306,30 @@ library AccountAccessParser {
         return keccak256(abi.encode(finalArray));
     }
 
+    function shouldIncludeDiff(address account, StateDiff memory diff, address _parentMultisig, bytes32 _txHash)
+        internal
+        view
+        returns (bool)
+    {
+        // 1. If the state change is an EOA nonce increment, remove it.
+        if (isEOANonceIncrement(account, diff)) {
+            return false;
+            // 2. Remove Gnosis Safe nonce increment and approve hash changes.
+        } else if (isGnosisSafe(account)) {
+            // 2.1 Nonce increment or 2.2 Setting an approve hash in storage.
+            if (isGnosisSafeNonceIncrement(diff) || isGnosisSafeApproveHash(diff, _parentMultisig, _txHash)) {
+                return false;
+            }
+            // 3. If the slot contains a timestamp, normalize it to zeroes.
+        } else if (slotContainsTimestamp(account, diff)) {
+            diff = normalizeTimestamp(diff);
+            // 4. If the slot is on the LivenessGuard, don't include it.
+        } else if (isLivenessGuardTimestamp(account, diff, _parentMultisig)) {
+            return false;
+        }
```

Comment:
nit, formatting of comments looks odd here, perhaps we should move them inside their `else if` blocks since putting the comment above makes it get indented

Reply:
https://github.com/ethereum-optimism/superchain-ops/pull/922/commits/7f0898394f0bea4d1f3979e97b1070577ac837d8